earth_enterprise/src/fusion/autoingest/StorageManager_unittest.cpp

```diff
@@ -56,10 +56,10 @@ class TestItem : public StorageManaged, public TestItemStorage {
 
   // determine amount of memory used by TestItem
   uint64 GetSize() {
-    return (GetObjectSize(val)
-    + GetObjectSize(type)
-    + GetObjectSize(savename)
-    + GetObjectSize(saveSucceeds));
+    return GetHeapUsage(val)
```

Comment:
Technically there's some things missing here (`loadname` and the stuff in StorageManaged) but I'm not sure it matters for unit tests.